src/actions/index.ts

```diff
@@ -58,6 +61,11 @@ const ACTION_TYPE_INDEX: {[k in ActionType]: 1} = {
   FILTER_MODIFY_ONE_OF: 1,
   FILTER_REMOVE: 1,
 
+  LOG_ERRORS_ADD: 1,
+  LOG_ERRORS_REMOVE: 1,
```

Comment:
When is error moved and warnings not removed? 

I wonder if we should just have a single `LOG_CLEAR` action?